agent/fw_wordpress.c

```diff
@@ -471,12 +506,17 @@ NR_PHP_WRAPPER(nr_wordpress_exec_handle_tag) {
   tag = nr_php_arg_get(1, NR_EXECUTE_ORIG_ARGS TSRMLS_CC);
 
   if (1 == nr_php_is_zval_non_empty_string(tag)
-      || (0 != NRINI(wordpress_hooks))) {
+      && (0 != NRINI(wordpress_hooks))) {
```

Comment:
This check was changed to match that of `nr_wordpress_apply_filters`. I believe this to be the correct functionality, as we likely don't ever want to be setting the hook tag if the INI is false and also want to prevent potential NULL tags if the INI is true.

The downstream code is safe for NULL tags, and just skips instrumentation if NULL. So this change could instrument more, as it prevents NULL skip from ever happening (while within a `nr_wordpress_exec_handle_tag` call stack).